deploy/helm/sumologic/values.yaml

```diff
@@ -765,10 +766,8 @@ prometheus-operator:
       interval:
   # Ensure we use pre 1.14 recording rules consistently as current content depends on them.
   kubeTargetVersionOverride: 1.13.0-0
-  ## Set the enabled flag to false for either of the below two purposes:
-  ## 1. Not install prometheus operator helm chart as a dependency along with this helm chart
-  ## 2. Disable metrics collection altogether
-  enabled: true
+  ## Set the enabled flag to false to not install prometheus operator helm chart as a dependency along with this helm chart
```

Comment:
```suggestion
## Uncomment the flag below to not install fluent-bit helm chart as a dependency along with this helm chart.
## Comment it out if you wish to have the fluent-bit dependency installed.
## Setting the flag to true instead of commenting out will break other functionality. 
```